src/display/annotation_layer.js

```diff
@@ -876,7 +876,10 @@ class CheckboxWidgetAnnotationElement extends WidgetAnnotationElement {
     const data = this.data;
     const id = data.id;
     const value = storage.getOrCreateValue(id, {
-      value: data.fieldValue && data.fieldValue !== "Off",
+      value:
```

Comment:
It should be enough to have `data.fieldValue === data.exportValue`.

Reply:
I first thought too, but the data.exportValue is defined in [https://github.com/mozilla/pdf.js/blob/6249ef517d3aaacc9aa6c9e1f5377acfaa4bc2a7/src/core/annotation.js#L1923](here). So there might be cases where this can be undefined?!

Reply:
Hi @calixteman . Do you think this should be enough because the pdf might be broken if the exportValue is not set? Should I change it according to your suggestion?

Reply:
It's likely better to be defensive so +1.